src/components/PreviewProcessedData.jsx

```diff
@@ -0,0 +1,9 @@
+const PreviewProcessedData = ({data}) => {
```

Comment:
Let's implement here a preview for the future downloaded file or files. How do you like this idea? The user may not want to download the file to see the result. 

How do u like such idea?

Reply:
WANT THIS !!!!

Reply:
but it sounds like smth that isn't easy...

Reply:
First of all I want to be realized opportunity to see all output data witout downloading

Place all the sheets in order under the control buttons. And let the general sheet be at the top.

And dont forget that preview should be the same structure as downloaded file/files

Reply:
I'll create new pull request for this task